lib/rubocop/cop/layout/initial_indentation.rb

```diff
@@ -26,18 +26,18 @@ def first_token
 
         def space_before(token)
           return unless token
-          return if token.pos.column.zero?
+          return if token.column.zero?
 
-          token_with_space =
+          space_range =
             range_with_surrounding_space(range: token.pos,
                                          side: :left,
                                          newlines: false)
           # If the file starts with a byte order mark (BOM), the column can be
           # non-zero, but then we find out here if there's no space to the left
           # of the first token.
-          return if token_with_space == token.pos
+          return if space_range == token.pos
```

Comment:
This is the only change outside of the scope of the summary. This previously named `token_with_space` is not a token at all. It is a `Parser::Source::Range`, so I renamed it.